pom.xml

```diff
@@ -184,7 +184,6 @@
       <groupId>org.apache.apex</groupId>
       <artifactId>apex-engine</artifactId>
       <version>${apex.core.version}</version>
-      <scope>test</scope>
```

Comment:
This is required as AbstractKeyValueStorageAgent.java needs access to com.datatorrent.stram.plan.logical and test scope would cause compilation failure.

Please check - incubator-apex-malhar/library/src/main/java/com/datatorrent/lib/util/AbstractKeyValueStorageAgent.java:[229,34]